tests/manage/cluster/nodes/test_disk_failures.py

```diff
@@ -5,14 +5,15 @@
 from ocs_ci.framework import config
 from ocs_ci.ocs.exceptions import CommandFailed
 from ocs_ci.ocs.resources.pod import get_all_pods
-from ocs_ci.framework.testlib import tier4, ignore_leftovers, ManageTest
+from ocs_ci.framework.testlib import tier4, ignore_leftovers, ManageTest, aws_platform_required
 from tests.sanity_helpers import Sanity
 from tests.helpers import wait_for_resource_count_change, get_admin_key
 
 logger = logging.getLogger(__name__)
 
 
 @tier4
+@aws_platform_required
```

Comment:
I don't think this is needed. This test was adjusted to work with any platform

Reply:
I guess we dont have lib written for vmware [liburl](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/platform_nodes.py#L80-L129)

Reply:
No, not yet

Reply:
I mean, it's not implemented in https://github.com/red-hat-storage/ocs-ci/blob/1b1293d2b0c667fbe40ef8964c28897b20709b35/ocs_ci/ocs/platform_nodes.py#L80.
I'm planning to do that in the upcoming days :)

Reply:
thanks, I removed aws_platform_required marker